### ISSUE TITLE
Crash on video options gui

### ISSUE THREAD

**Issue by Ironmango21** [coolsquid/MiscTweaks#3]
https://pastebin.com/9b40t8Wu

The game crashes upon opening video options, whether in game or from the main menu. It seems to be an issue with one of the config oprions, gamma maybe? Starting the game with a blank config fixes the issue temporarily.
Caused by: java.lang.ClassCastException: [Lnet.minecraft.client.settings.GameSettings$Options; cannot be cast to net.minecraft.client.gui.GuiOptionsRowList

**Comment by coolsquid** [comment]
Yes, that is an issue with the gamma option. I have identified the cause, and will look for a fix. Until then, disable that option.
